index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Dev Calculator Design</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="calculator">
+        <div class="calculator--screen">
+            <h1>9-2</h1>
+        </div>
+        <div class="calculator--buttons">

```

Comment:
Hi David, good job so far. You should use buttons for your calculator buttons. See if you can refactor this.